src/ray/raylet/node_manager.h

```diff
@@ -696,7 +705,7 @@ class NodeManager : public rpc::NodeManagerServiceHandler,
   ///
   /// \param Output parameter. `resource_load` and `resource_load_by_shape` are the only
   /// fields used.
-  void FillResourceUsage(std::shared_ptr<rpc::ResourcesData> data) override;
+  void FillResourceUsage(rpc::ResourcesData *data) override;
```

Comment:
Why change to a raw ptr?

Reply:
Because grpc gives raw pointers (like `request_resource_report_reply->mutable_resources()`)